demos/mpc-as-a-service/client-mpc.js

```diff
@@ -29,16 +31,26 @@
       jiff_instance = jiff_instance_;
     }
 
+    // Set up array sharing
+    let skeleton = jiff_instance.skeleton_of(input);
+    let skeletons = {};
+    for (var i = 0; i < config.input_parties.length; i++) {
+      let p_id = config.input_parties[i];
+      skeletons[p_id] = skeleton;  // Assume same skeleton for all parties
+    }
+
     // Share with compute parties
-    jiff_instance.share(input, null, config.compute_parties, config.input_parties);
+    jiff_instance.share_ND_array(input, skeletons, null, config.compute_parties, config.input_parties);
 
     // If this party is still connected after the compute parties are done, it will
     // receive the result.
-    var promise = jiff_instance.receive_open(config.compute_parties);
-    promise.then(function () {
+
+    var promise = jiff_instance.open_ND_array(null, all_parties, config.compute_parties);
```

Comment:
interesting that we can do open_ND_array(null) and it works. there was some reason why this did not work for regular open, and I had to do receive_open separately. 

We should make sure this is well documented. It would be nice to have the same feel to the API of open_ND_array and open
@p-flock what do you think?

Reply:
Ok, I see that we have "receive_open_array " as a bound alias to this.

let us use that here instead.